cookbooks/add_memory_using_qdrant_cloud.py

```diff
@@ -0,0 +1,40 @@
+# This example shows how to use vector config to use QDRANT CLOUD
+import os
+from dotenv import load_dotenv
+from mem0 import Memory
+
+# Loading my API_KEY for OPENAI
+load_dotenv()
+OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+USER_ID = "rajib"
```

Comment:
Hey, please can you keep a generic name here?

Reply:
where do you want me to put generic name

Reply:
Instead of `rajib` you can do `test`.

Reply:
done